openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -135,68 +135,18 @@ Both sending Credential Offer same-device and cross-device is supported.
-  * `software`: It MUST be used when the Wallet uses software-based key management.
-  * `hardware`: It MUST be used when the wallet uses hardware-based key management.
-  * `tee`: It SHOULD be used when the Wallet uses the Trusted Execution Environment for key management.
-  * `secure_enclave`: It SHOULD be used when the Wallet uses the Secure Enclave for key management.
-  * `strong_box`: It SHOULD be used when the Wallet uses the Strongbox for key management.
-  * `secure_element`: It SHOULD be used when the Wallet uses a Secure Element for key management.
-  * `hsm`: It SHOULD be used when the Wallet uses Hardware Security Module (HSM).
-* `user_authentication`: OPTIONAL. JSON String that asserts the security mechanism the Wallet uses to authenticate the user to authorize access to the private key associated with the public key given in the `cnf` claim. This specification defines the following values for `user_authentication`:
-  * `system_biometry`: It MUST be used when the key usage is authorized by the mobile operating system using a biometric factor.
-  * `system_pin`: It MUST be used when the key usage is authorized by the mobile operating system using personal identification number (PIN).
-  * `internal_biometry`: It MUST be used when the key usage is authorized by the Wallet using a biometric factor.
-  * `internal_pin`: It MUST be used when the key usage is authorized by the Wallet using PIN.
-  * `secure_element_pin` It MUST be used when the key usage is authorized by the secure element managing the key itself using PIN.
-
-The Wallet Attestation MAY also contain the following claim:
-
-* `aal`: OPTIONAL. JSON String asserting the authentication level of the Wallet and the key as asserted in the `cnf` claim.
-
-To obtain the issuer's Public key for verification, wallet attestions MUST support web-based key resolution as defined in Section 5 of [@!I-D.ietf-oauth-sd-jwt-vc]. The JOSE header `kid` MUST be used to identify the respective key.
-
-This is an example of a Wallet Instance Attestation:
-
-```json
-{
-  "typ": "wallet-attestation+jwt",
-  "alg": "ES256",
-  "kid": "1"
-}
-.
-{
-  "iss": "<identifier of the issuer of this wallet attestation>",
-  "sub": "<`client_id` of the OAuth client>",
-  "iat": 1516247022,
-  "exp": 1541493724,
-  "aal" : "https://trust-list.eu/aal/high",
-  "cnf": {
-    "jwk": {
-      "kty": "EC",
-      "crv": "P-256",
-      "x": "TCAER19Zvu3OHF4j4W4vfSVoHIP1ILilDls7vCeGemc",
-      "y": "ZxjiWWbZMQGHVWKVQ4hbSIirsVfuecCE6t4jT9F2HZQ"
-    },
-    "key_type": "strong_box",
-    "user_authentication": "system_pin",
-  }
-}
-```
+### Wallet Attestation {#wallet-attestation}
+
+Wallets MUST use wallet attestations as defined in Annex E of [@!OIDF.OID4VCI].
+
+To obtain the issuer's Public key for verification, Wallet Attestions MUST support web-based key resolution as defined in Section 5 of [@!I-D.ietf-oauth-sd-jwt-vc]. The JOSE header `kid` MUST be used to identify the respective key.
```

Comment:
during the WG call, agreed to mandate `x5c` only as a starting point.

Reply:
```suggestion
The issuer of the Wallet Attestation MUST use x509 certificates to sign and convey trust information. The JOSE header `x5c` MUST be used to identify the respective key.
```

Reply:
```suggestion
To obtain the issuer's Public key for verification, Wallet Attestions MUST support x.509 certificates. The JOSE header `x5c` MUST be used to identify the respective certificate.
```

Reply:
```suggestion
The Wallet Attestation MUST contain the X.509 certificate of the issuer along with an optional trust chain in the `x5c` JOSE header. The public key contained X.509 certificate MUST be used to verify the signature of the Wallet Attestation JWT.
```

Reply:
I dislike "obtain", as we are not obtaining anything with x5c

Reply:
```suggestion
The public key, and optionally a trust chain, used to validate the signature on the Wallet Attestation MUST be included in the `x5c` JOSE header.
```